edgedb-errors/src/error.rs

```diff
@@ -171,6 +177,9 @@ impl fmt::Display for Error {
                 write!(f, "{}", kind)?;
             }
         }
+        if let Some((line, col)) = self.line().zip(self.column()) {
+            write!(f, " (on line {}, column {})", line, col);
```

Comment:
I had added a comment to this line, but somehow the comment doesn't seem to be there anymore. Here it is again:

On this line `Result` isn't handled (triggering a compiler warning).

Reply:
It's visible, all fine. Thanks!